internal/auth/auth.go

```diff
@@ -0,0 +1,125 @@
+package auth
+
+import (
+	"html/template"
+	"net/http"
+	"sync"
+	"time"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+var Templates = template.Must(template.ParseGlob("../../internal/public/*.html"))
```

Comment:
Make sure that internal public *.html is embeddedable as a filesystem. Otherwise we have to copy the folder into the container. Note that skaffold build does not currently do this.